### ISSUE TITLE
NNI doesn't import ae sub interfaces

### ISSUE THREAD

**Issue by XioNoX** [Anthony25/netbox-netprod-importer#7]
Subinterfaces such as ae2.2002, ae3.2003, etc.. are not being imported into Netbox and their IPs appear under the parent ae interface.
This is at least on Juniper MX routers running 13.3.

Please let me know how I can help troubleshoot it.

**Comment by Anthony25** [comment]
Hi,
Thanks a lot for your bugreport. I'm going to dig a bit, I don't remember if I did this on purpose or not (and let the user tag the parent interfaces into the right vlan instead of creating the subinterfaces).

**Comment by Anthony25** [comment]
Yes, I did this on purpose: https://github.com/Anthony25/netbox-netprod-importer/blob/master/netbox_netprod_importer/importer.py#L143

Would you like an option to import the subinterfaces?

**Comment by XioNoX** [comment]
Ideally yeah, Netbox should see every configurable interfaces of the device, sub-interfaces included (at least for our usecase).
